### ISSUE TITLE
[1.3.x] Creating item in 1:1 relation failed on source side

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#838]
```
Fatal error: Call to a member function setProductClass() on a non-object in /domain/modules/Helper/lib/Helper/Entity/Base/AbstractProductClass.php on line 389
```
Settings:

- bidirectional
- cascade for both directions: persist

**Comment by Guite** [comment]
Postponed to 0.7.1: the issue does not block the module of @portugao since he made the association unidirectional. The problem does not affect unidirectional relationships, only bidirectional ones.

**Comment by robbrandt** [comment]
Just a heads up in case 0.70 Final is imminent...  I also experienced a relational issue of some sort with a One to Many bidirectional on 1.3.x; I created a child record but no relationship created.  I haven't had time to figure out whether I made a model error or if it's a bug.

And.... this 1.3.x issue is postponed to 0.71, but 1.3.x support will be dropped after 0.70 Final?  Did I miss something?
